src/components/seo.js

```diff
@@ -72,15 +73,21 @@ function SEO({ description, lang, meta, title, imageUrl }) {
         },
         {
           name: `twitter:card`,
-          content: `summary`,
+          content: `summary_large_image`,
+        },
+        {
+          name: `twitter:image`,
+          content: metaImageUrl.includes("http")
+            ? metaImageUrl
+            : `https://theyworkforus.elect.in.th${metaImageUrl}`,
```

Comment:
maybe use `withPrefix` for the url? If we use it, we also have to update the build command as well.

https://www.gatsbyjs.org/docs/path-prefix/

@rapee  what do you think?

Reply:
To my understanding, `withPrefix`, as the name suggested, adds only prefix path, not domain. I think it's not our solution?

We can build using `BASE_URL=https://theyworkforus.elect.in.th` and pass it from `gatsby-config.js` (build step) to `seo.js` (frontend).

Reply:
we could also create a similar function using BASE_URL.

Reply:
Saw the usage of [location props](https://www.gatsbyjs.org/docs/location-data-from-props/) and came up with creating dynamic `<meta>` based on `location.origin` (and `withPrefix`). But I have tested and learned that it is not feasible. Fetchers surely don't wait for that. Silly me.

So, I added a `baseUrl` in `siteMetadata` which is configurable through `BASE_URL`. Don't forget to update any CI/CD automation 🥂

Reply:
The **location props** you mentioned I believe is working only client side i.e. live site when user is visiting. I believe `<meta>` is populated during the build process so we'll need full URL in this step.